server/statistics/hot_peer_cache.go

```diff
@@ -377,7 +377,7 @@ func (f *hotPeerCache) getDefaultTimeMedian() *movingaverage.TimeMedian {
 }
 
 func (f *hotPeerCache) updateHotPeerStat(newItem, oldItem *HotPeerStat, bytes, keys float64, interval time.Duration) *HotPeerStat {
-	if newItem == nil || newItem.needDelete {
+	if newItem.needDelete {
```

Comment:
so it won't be nil?

Reply:
Before we invoke this function, we will create a `newItem` and it cannot be nil in the current implementation.